system-tests/docker-test-env/styx-config/styxconf.yml

```diff
@@ -4,5 +4,8 @@ include: /styx/default-config/default.yml
 services:
   factories:
     backendServiceRegistry:
+      class: "com.hotels.styx.proxy.backends.file.FileBackedBackendServicesRegistry$Factory"
       config:
-        originsFile: "/styx/config/origins.yml"
+        originsFile: "/styx/default-config/origins.yml"
```

Comment:
Should this be /styx/config/origins.yml as before?

Reply:
Fixed.